docs/core/tools/enable-tab-autocomplete.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Enable tab completion .NET Core CLI
+description: This article teaches you how to enable tab completion for the .NET Core CLI in v2.0 and above for PowerShell, Bash, and zsh.
+author: thraka
+ms.author: adegeo
+ms.date: 12/17/2018
+---
+
+# How to enable TAB completion for the .NET Core CLI
+
+Starting with .NET Core CLI v2.0, support for tab completion is included but disabled by default. This article describes how to enable tab completion for your terminal of choice.
+
+[!INCLUDE topic-appliesto-net-core-2plus]
+
+Tab completion for .NET Core is triggered by typing a `dotnet ` command in the terminal, and then hitting the TAB key. The current command line is sent to the `dotnet complete` command, and the results are processed by your terminal. You can test the results without enabling tab completion by sending something directly to the `dotnet complete` command. For example:
+
+```
+> dotnet complete "dotnet a"
+add
+clean
+--diagnostics
+migrate
+pack
+```
+
+If that command doesn't work, make sure that .NET Core 2.0 or above is installed. If it's installed, but that command still doesn't work, make sure that the `dotnet` command resolves to a version of .NET Core 2.0 and above.
+
+> [!TIP]
+> Use the `dotnet --version` command to see what version of `dotnet` your current path is resolving to. For more information, see [Select the .NET Core version to use](../versions/selection.md).
+
+
+### Examples
+
+Here are some examples of what tab completion provides:
+
+Input                                | becomes                                                                     | because
+:------------------------------------|:----------------------------------------------------------------------------|:--------------------------------
+`dotnet a⇥`                          | `dotnet add`                                                                 | `add` is the first subcommand, alphabetically.
+`dotnet add p⇥`                      | `dotnet add --help`                                                          | Tab completion matches substrings and `--help` comes first alphabetically.
+`dotnet add p⇥⇥`                    | `dotnet add package`                                                          | Pressing tab a second time brings up the next suggestion.      
+`dotnet add package Microsoft⇥`      | `dotnet add package Microsoft.ApplicationInsights.Web`                      | Results are returned alphabetically.
+`dotnet remove reference ⇥`          | `dotnet remove reference ..\..\src\OmniSharp.DotNet\OmniSharp.DotNet.csproj` | Tab completion is project file aware.
+
+## PowerShell
+
+To add tab completion to **PowerShell** for the .NET Core CLI, add the following code to your PowerShell profile stored in `$PROFILE`:
```

Comment:
how do you access this?
when I type notepad $PROFILE on PS, I get an error

Reply:
I'll edit this to say create/edit the profile stored in the $PROFILE variable. Your error is just due to the fact that the file doesn't exist unless you create it.